src/sentry/integrations/slack/endpoints/action.py

```diff
@@ -194,6 +195,15 @@ def post(self, request: Request) -> Response:
         user_id = slack_request.user_id
         response_url = data.get("response_url")
 
+        if data.get("actions") and data["actions"][0].get("value", "") in ["link", "ignore"]:
+            payload = {"delete_original": "true"}
+            try:
+                post(response_url, json=payload)
+            except ApiError as e:
+                logger.error("slack.action.response-error", extra={"error": str(e)})
```

Comment:
Should we match the pattern of the other actions here and `self.respond(status=403)` on an error?